### DISCUSSION TITLE
Checklist Editor: Show headers

### DISCUSSION THREAD

**Discussion by vHanda** [GitJournal/GitJournal#827] [Feature Requests]
Currently the Checklist editor only shows task lists, and ignores all other markdown. Ideally it would be great if we could show the rest of the markdown, but since that's challenging, maybe we could just start with the headers.

---
> Please vote on this issue with a 👍reaction. It's one of the primary mechanisms used to prioritize issues.

**Comment by akvadrako** [comment]
Would it be similarly easy to add checklist-editing to the markdown viewer? I mean that checklists in the viewer gain the features of the checklist editor: larger, tickable boxes and edit/add functions.

**Comment by vHanda** [comment]
> Would it be similarly easy to add checklist-editing to the markdown viewer? I mean that checklists in the viewer gain the features of the checklist editor: larger, tickable boxes and edit/add functions.

Sadly, that's challenging. It's just cause of the way dart's markdown parser is implemented and how Flutter's markdown viewer operates.
